testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -299,6 +300,28 @@ def test_box(self, universe, index, expected):
         assert_almost_equal(self.box_refs[expected], universe.dimensions)
 
 
+class TestNCDFReader4(object):
+    """NCDF Trajectory exported by cpptaj, without `time` variable.
+    """
+    prec = 3
+
+    @pytest.fixture(scope='class')
+    def u(self):
+        return mda.Universe(CPPTRAJ_TRAJ_TOP,
+                            [CPPTRAJ_TRAJ, CPPTRAJ_TRAJ])
+
+    def test_chain_times(self, u):
+        """Check times entries for a chain of trajectories without a defined time variable""
```

Comment:
```suggestion
        """Check times entries for a chain of trajectories without a defined time variable"""
```
Missing an extra quote mark.